src/domains/transaction/validations/UsernameRegistration.ts

```diff
@@ -42,7 +42,11 @@ export const usernameRegistration = (t: any) => ({
 			unique: debounceAsync(async (value) => {
 				try {
 					await usernameExists(network, value, controller);
-				} catch {
+				} catch (error) {
+					if (error && error.name === "AbortError") {
+						return true;
+					}
+
 					return t("COMMON.VALIDATION.EXISTS", { field: t("COMMON.USERNAME") });
```

Comment:
wouldn't it be better to check for the error you expect instead and then return it as "already exists" instead of ignoring one specific error while others may still cause false positives? 🤔